substate/message.go

```diff
@@ -11,6 +11,14 @@ import (
 	"github.com/0xsoniclabs/substate/types/hash"
 )
 
+// Transaction types.
+const (
+	LegacyTxType     = 0x00
+	AccessListTxType = 0x01
+	DynamicFeeTxType = 0x02
+	BlobTxType       = 0x03
+)
```

Comment:
nit: use iota for incremented constants in golang https://go.dev/wiki/Iota

```suggestion
const (
	LegacyTxType = iota
	AccessListTxType
	DynamicFeeTxType
	BlobTxType
)
```

Reply:
also, add the `SetCodeTxType` even tho it wont used, I believe we should have it here

Reply:
Oh ic. Yeah lets keep it that way